junit-platform-jfr/src/module/org.junit.platform.jfr/module-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+/**
+ * Defines the JUnit Platform JAR API.
+ *
+ * @since 1.7
+ */
+module org.junit.platform.jfr {
+	requires jdk.jfr;
+	requires transitive org.apiguardian.api;
+	requires org.junit.platform.commons;
```

Comment:
I don't see commons in the dependencies in the buildscript nor used in the source code. Can that be removed from here?

Reply:
Think so, yes.

Also the `transitive` modifiers are not needed here -- as no package is exported.